neo/SmartContract/Native/Votes/Method/FindAllPathMethod.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract.Native
+{
+    public class FindAllPathMethod
+    {
+        /// <summary>
+        /// Node Set
+        /// </summary>
+        private List<String> nodesList = new List<String>();
+        /// <summary>
+        /// Vector Graphic,describe the connection between nodes and nodes
```

Comment:
between nodes and nodes? Maybe just between nodes. It describes the vertices and the edges it connects to?

Reply:
ping @doubiliu @Lichen9618